tests/experimental/datasources/test_pandas_s3_datasource.py

```diff
@@ -55,16 +56,31 @@ def aws_s3_bucket_name() -> str:
     return "test_bucket"
 
 
-def _get_boto3_client(aws_region_name: str, aws_s3_bucket_name) -> BaseClient:
-    conn = boto3.resource("s3", region_name=aws_region_name)
-    conn.create_bucket(Bucket=aws_s3_bucket_name)
-    client = boto3.client("s3", region_name=aws_region_name)
-    return client
+@pytest.fixture(scope="function")
+def aws_credentials() -> None:
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
 
 
-def _build_pandas_s3_datasource(
-    aws_region_name: str, aws_s3_bucket_name
-) -> PandasS3Datasource:
+@pytest.fixture
+def s3(aws_credentials, aws_region_name: str) -> BaseClient:
```

Comment:
```suggestion
def s3_mock(aws_credentials, aws_region_name: str) -> BaseClient:
```
a fixture called `s3` is very ambiguous

Reply:
@Kilo59 Good point -- fixed.